Dockerfile

```diff
@@ -1,16 +1,13 @@
 FROM node:lts
 
-WORKDIR /app
+RUN npm i -g nodemon typescript
 
-RUN chown node:node /app
+USER node
 
-COPY . .
+WORKDIR /app
 
-RUN npm install -g nodemon
-RUN npm install typescript -g
-RUN npm install -g
+COPY --chown=node . .
 
-USER node
+RUN npm i
 
-EXPOSE 8100
-RUN npm run dev
\ No newline at end of file
+CMD npm run migrate && npm run dev
```

Comment:
We add confusing with actual Dockerfile that is used to dev.
I will be better to have a production ready Dockerfile and a specific docker file for dev (like "Dockerfile.dev")

Reply:
happy to take this on in the next PR